### PR TITLE
feature: add quick-start script

### PR DESCRIPTION
**Pull request by snakorse**
#### What type of this PR

Add one of the following kinds:
/kind document

#### Specified Reviewers:

/assign @luobily

### CODE REVIEW

File: quick-start/quick-start.sh [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+if ! command -v docker-compose >/dev/null; then
+    abort "$(cat <<EOABORT
+You must install Docker-Compose before installing Erda.
+EOABORT
+)"
+fi
+
+INSTALL_LOCATION="/opt/erda-quickstart"
+ERDA_VERSION="1.2.1"
```
Comment (iutx) [inline quick-start/quick-start.sh:156]: please check erda version.
current master erda version is 1.4, 1.5 coming soon.

**Review by iutx — COMMENTED** [review]
@snakorse cc

File: quick-start/quick-start.sh (hunk 1 above)
Comment (snakorse) [reply quick-start/quick-start.sh]: done, this value is just a placeholder, ERDA_VERSION will got rendered with the latest supported version.

**Pull request merged**